packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -244,6 +244,38 @@ export class MgtPerson extends MgtTemplatedComponent {
   })
   public avatarSize: AvatarSize;
 
+  /**
+   * object containing Graph details on person
+   * a copy of person-details attribute
+   * @type {IDynamicPerson}
+   */
+  @property({
+    attribute: null,
+    type: Object
+  })
+  private get personDetailsInternal(): IDynamicPerson {
```

Comment:
can you comment somewhere on why the internal version needs to exist and the differentiation with persondetails.